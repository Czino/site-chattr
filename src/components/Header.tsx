import { getDisplayUrl } from '../helpers/getDisplayUrl'

type Props = { url: string }
export const Header = ({ url }: Props) => (
    <header className="sticky top-0 p-2 text-center bg-purple-400">
        <h1 className="text-purple-50">SiteChattr</h1>
        <h2 className="overflow-x-auto text-purple-50">{getDisplayUrl(url)}</h2>
    </header>
)
