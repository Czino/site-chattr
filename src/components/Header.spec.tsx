import { render } from '@testing-library/react'
import { Header } from './Header'

describe('Header', () => {
    it('renders correctly', () => {
        const { asFragment } = render(<Header url="https://localhost" />)
        expect(asFragment()).toMatchSnapshot()
    })
})
