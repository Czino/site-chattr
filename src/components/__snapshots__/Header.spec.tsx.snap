// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <header
    class="sticky top-0 p-2 text-center bg-purple-400"
  >
    <h1
      class="text-purple-50"
    >
      SiteChattr
    </h1>
    <h2
      class="overflow-x-auto text-purple-50"
    >
      localhost/
    </h2>
  </header>
</DocumentFragment>
`;
